home-work-app/src/app/components/MoviesSection/MoviesSection.tsx

```diff
@@ -16,54 +16,54 @@ interface MoviesSectionProps extends MapDispatchToProps {
     moviesCollection: IApiResponse.IMovie;
 }
 
-const filterItems: FilterItems = AppConstants.filterItems;
+const { filterItems } = AppConstants;
 
-const mapStateToProps: MapStateToProps = (state: GlobalState): {
-    moviesCollection: IApiResponse.IMovie[]
+const mapStateToProps: MapStateToProps = (
+    state: GlobalState
+): {
+    moviesCollection: IApiResponse.IMovie[];
 } => ({
-    moviesCollection: state.movieEditor.movies
+    moviesCollection: state.movieEditor.movies,
 });
 
 const mapDispatchToProps: MapDispatchToProps = {
     deleteMovie: deleteMovieAction,
-    update: updateMovieAction
+    update: updateMovieAction,
 };
 
 const MoviesSectionComponent: React.FC<MoviesSectionProps> = ({
     moviesCollection,
     deleteMovie,
-    update
+    update,
 }: MoviesSectionProps) => {
     const [movies, setMovies] = useState(null);
     useEffect(() => {
-            update();
-            setMovies(moviesCollection);
+        update();
+        setMovies(moviesCollection);
     }, [moviesCollection]);
 
     return (
         <MoviesListStyled>
             <FilterPanel filterItems={filterItems} />
```

Comment:
You should pass current sorting and filter option from `redux` to `FilterPanel` via props.